docs/UserGuide.md

```diff
@@ -394,9 +420,33 @@ data file at the next run. Hence, it is recommended to take a backup of the file
 
 ## FAQ
 
-Q: How do I transfer my data to another Computer?
-A: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the
-data of your previous A&E home folder.
+**Q**: I double-clicked the Jar file and it doesnt run, why?\
+**A**: Make sure that Java 11 is installed and do refer to step 5 of [Quick-start](#quick-start) for instructions to run
+application.
+
+**Q**: I have a list of patients, how do i view their details?\
+**A**: Please click on the patient you are interested in viewing. The details will be on the right hand side in the
+Patient Record Panel.
+
+**Q**: The Patient List Panel is cut off, i cant see some details for each patient\
+**A**: Please resize the application by dragging the border of the application window with your mouse.
+If the appplication is already in full screen mode, please drag the divider between the Patient List Panel and Patient
+Record Panel to the right.
+
+**Q**: I entered a command and there is no response from the application, what do i do?\
+**A**: A critical error might have occured. Please ensure your command is as specified in [Features](#features). Please contact
+the technical team in charge of your department to file a bug report on our [issues](tps://github.com/AY2324S1-CS2103T-T14-2/tp/issues) page.
+
+**Q**: I have accidentally cleared the whole patient record system, can i retrieve the lost details?\
+**A**: Yes, A&E has the [Undo](#undoing-a-command--undo) feature which allows you to retrieve all the lost details.
+
+**Q**: Will my data be saved immediately?\
+**A**: Yes, A&E saved your data after every command executed.
+
+**Q**: How do I transfer my data to another Computer?\
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains
```

Comment:
Perhaps we could be clearer on this "data file" part in case users do not understand what it is and how to overwrite it?